docs/source/cpp/parquet.rst

```diff
@@ -27,15 +27,203 @@ Reading and writing Parquet files
+
++--------------------------+-------------------------+---------+
+| Physical type            | Mapped Arrow type       | Notes   |
++==========================+=========================+=========+
+| BOOLEAN                  | Boolean                 |         |
++--------------------------+-------------------------+---------+
+| INT32                    | Int32 / other           | \(1)    |
++--------------------------+-------------------------+---------+
+| INT64                    | Int64 / other           | \(1)    |
++--------------------------+-------------------------+---------+
+| INT96                    | Timestamp (nanoseconds) | \(2)    |
++--------------------------+-------------------------+---------+
+| FLOAT                    | Float32                 |         |
++--------------------------+-------------------------+---------+
+| DOUBLE                   | Float64                 |         |
++--------------------------+-------------------------+---------+
+| BYTE_ARRAY               | Binary / other          | \(1)    |
++--------------------------+-------------------------+---------+
+| FIXED_LENGTH_BYTE_ARRAY  | FixedSizeBinary / other | \(1)    |
++--------------------------+-------------------------+---------+
+
+* \(1) Can be mapped to other Arrow types, depending on the logical type
+  (see below).
+
+* \(2) On the write side, :func:`ArrowWriterProperties::support_deprecated_int96_timestamps`
+  must be enabled.
+
+Logical types
+~~~~~~~~~~~~~
+
+Specific logical types can override the default Arrow type mapping for a given
+physical type.  If the Parquet file contains an unrecognized logical type,
+the default physical type mapping is used.
+
++-------------------+-----------------------------+----------------------------+---------+
+| Logical type      | Physical type               | Mapped Arrow type          | Notes   |
++===================+=============================+============================+=========+
+| NULL              | Any                         | Null                       | \(1)    |
++-------------------+-----------------------------+----------------------------+---------+
+| INT               | INT32                       | Int8 / UInt8 / Int16 /     |         |
+|                   |                             | UInt16 / Int32 / UInt32    |         |
++-------------------+-----------------------------+----------------------------+---------+
+| INT               | INT64                       | Int64 / UInt64             |         |
++-------------------+-----------------------------+----------------------------+---------+
+| DECIMAL           | INT32 / INT64 / BYTE_ARRAY  | Decimal128 / Decimal256    |         |
+|                   | / FIXED_LENGTH_BYTE_ARRAY   |                            |         |
++-------------------+-----------------------------+----------------------------+---------+
+| STRING            | BYTE_ARRAY                  | Utf8                       |         |
```

Comment:
Do you support `LargeUtf8`?

Reply:
Yes, it is. So is `LargeBinary`. I'll add a note, thanks.